### PR TITLE
Feat/max timestamp ride creation

### PR DESCRIPTION
**Pull request by Scttpr**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Scttpr** [comment]
[Ne pas permettre la création de course coté soutenu sur une temporalitée configurable](https://trello.com/c/wyecKmUv/231-ne-pas-permettre-la-cr%C3%A9ation-de-course-cot%C3%A9-soutenu-sur-une-temporalit%C3%A9e-configurable)

File: middlewares/mask-output.mjs [hunk 1]
```diff
@@ -26,6 +26,24 @@ export function cleanObject(e, ctx) {
...
+          toAdd = false;
+        }
+      });
+      if (toAdd) {
+        newAcc.push(item);
+      }
+    });
+    return newAcc;
+  }, [])
+  .join(',');
```
Comment (Nainterceptor) [inline middlewares/mask-output.mjs]: This behavior introduce infosec issue

File: models/campus.mjs [hunk 2]
```diff
@@ -44,6 +44,10 @@ const CampusSchema = new Schema({
     },
     default: 30,
   },
+  defaultReservationScope: {
```
Comment (Nainterceptor) [inline models/campus.mjs]: Must be saved in the lower metric (for javascript, it's ms, but we can save time in seconds, ms don't really have sense)

File: routes/campuses.mjs [hunk 3]
```diff
@@ -45,7 +46,11 @@ const router = generateCRUD(Campus, {
       async (ctx, next) => {
         await next();
         if (!ctx.may(CAN_GET_CAMPUS)) {
-          ctx.body = mask(ctx.body, BASIC_OUTPUT_MASK);
+          const masks = [BASIC_OUTPUT_MASK];
+          if (ctx.request.query.mask) {
```
Comment (Nainterceptor) [inline routes/campuses.mjs]: As explained, introduce infosec issue

**Pull request merged**